packages/api-client/src/team/data/TeamMemberUpdateData.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Wire
+ * Copyright (C) 2020 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {PermissionsData} from '../member/PermissionsData';
+
+export interface TeamMemberUpdateData {
```

Comment:
Does it also contain `created_by` & `created_at` like in `MemberData`?

Reply:
No:

![Screenshot](https://user-images.githubusercontent.com/5497598/78350984-5f2c9680-75a6-11ea-8e43-8b3ffb7301f6.png)

Reply:
It's optional so does this user have these values?

Reply:
Mhh it's possible that it just returns `MemberData`..

Reply:
AFAIC the server returns only `user` and `permissions`: https://github.com/wireapp/wire-server/blob/fffbd95254cbd273646242af23feb30af3ea797b/libs/galley-types/src/Galley/Types/Teams.hs#L908

Reply:
Alright, thx for digging :)